docs/spelling_wordlist

```diff
@@ -780,7 +780,6 @@ viewable
 virtualized
 Weblog
 whitelist
-whitelisted
```

Comment:
Can `whitelist` also be removed here now? (If not why not?)

Reply:
https://docs.djangoproject.com/en/3.0/ref/validators/#django.core.validators.EmailValidator.whitelist